Tools/Cli-LoRa-Device-Provisioning/Cli-LoRa-Device-Provisioning.csproj

```diff
@@ -22,19 +22,24 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="CommandLineParser" Version="2.4.3" />
-    <PackageReference Include="Microsoft.Azure.Devices" Version="1.17.2" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.2.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
+    <PackageReference Include="CommandLineParser" Version="2.8.0" />
+    <PackageReference Include="Microsoft.Azure.Devices" Version="1.22.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.1.6" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
   </ItemGroup>
 
     <!-- StyleCop Setup -->
     <ItemGroup>
         <AdditionalFiles Include="../../stylecop.json" Link="stylecop.json" />
     </ItemGroup>
     <PropertyGroup>
-        <CodeAnalysisRuleSet>../../stylecop.ruleset</CodeAnalysisRuleSet><Authors>Sascha Corti (saschac@microsoft.com)</Authors><Version>1.0.0.0</Version><Company>Microsoft</Company><Product>Azure IoT Edge LoRaWAN Starter Kit LoRa Leaf Device Provisioning Utility.</Product><Description>This tool complements http://aka.ms/lora</Description><Copyright>Copyright (c) 2019 Microsoft. All rights reserved.</Copyright><PackageLicenseExpression>Licensed under the MIT license. See LICENSE file in the project GitHub repo root for full license information.</PackageLicenseExpression><PackageProjectUrl>https://github.com/Azure/iotedge-lorawan-starterkit</PackageProjectUrl><RepositoryUrl>https://github.com/Azure/iotedge-lorawan-starterkit</RepositoryUrl>
+        <CodeAnalysisRuleSet>../../stylecop.ruleset</CodeAnalysisRuleSet><Authors>Sascha Corti (saschac@microsoft.com)</Authors><Version>1.1.0.0</Version><Company>Microsoft</Company><Product>Azure IoT Edge LoRaWAN Starter Kit LoRa Leaf Device Provisioning Utility.</Product><Description>This tool complements http://aka.ms/lora</Description><Copyright>Copyright (c) 2019 Microsoft. All rights reserved.</Copyright><PackageLicenseExpression>Licensed under the MIT license. See LICENSE file in the project GitHub repo root for full license information.</PackageLicenseExpression><PackageProjectUrl>https://github.com/Azure/iotedge-lorawan-starterkit</PackageProjectUrl><RepositoryUrl>https://github.com/Azure/iotedge-lorawan-starterkit</RepositoryUrl>
     </PropertyGroup>
-    <Import Project="../../stylecop.props" />
+    <Import Project="../../stylecop.props" /><ItemGroup>
+  <PackageReference Update="StyleCop.Analyzers" Version="1.1.118">
```

Comment:
I think this is not needed as you reference another stylecop file

Reply:
Ok, removed.